internal/builder/builder.go

```diff
@@ -102,10 +102,21 @@ type moduleWithDiffID struct {
 type BuilderOption func(*options) error
 
 type options struct {
-	toFlatten buildpack.FlattenModuleInfos
+	flatten  bool
+	exclude  []string
+	runImage string
+  toFlatten buildpack.FlattenModuleInfos
 	labels    map[string]string
 }
 
+func WithRunImage(name string) BuilderOption {
+	return func(o *options) error {
+		o.runImage = name
+		return nil
+	}
+
```

Comment:
```suggestion
}
```

This should fix the linter

Reply:
@Pratham1812 could you do this fix?